src/api/category/index.ts

```diff
@@ -0,0 +1,8 @@
+import { api } from '..';
+
+export const getCategories = async () => {
+  const response = await api.get(
+    'https://port-0-pream-server-m2kkbe2ebf996543.sel4.cloudtype.app/categories'
+  );
+  return response;
```

Comment:
`api`는 axios 인스턴스인데, 여기에서 미리 base url 설정을 해 준 상태라 아래처럼만 작성해도 됩니다!
슬랙에 말씀드린 대로 `.env` 파일 추가해서 사용하시면 돼요!
```suggestion
  const response = await api.get('/categories');
  return response;
```

그리고 아래처럼 작성하면 추후에 react-query를 이용해서 데이터를 사용할 때 `data.data` 이렇게 사용하지 않고 `data`로만 바로 사용할 수 있습니다!
```suggestion
  const response = await api.get('/categories');
  return response.data;
```
```javascript
const { data } = useCategoryQuery();
const categories = data?.map((category) => category.name));
```

Reply:
수정하였습니다 :-)